mantidimaging/gui/windows/main/view.py

```diff
@@ -69,13 +69,13 @@ def show_save_dialogue(self):
         self.save_dialogue.show()
 
     def show_cor_tilt_window(self):
-        CORTiltDialogView(self).show()
+        CORTiltWindowView(self).show()
 
     def show_filters_window(self):
-        FiltersDialogView(self).show()
+        FiltersWindowView(self).show()
```

Comment:
Might there be a lifetime issue here now? I don't think `show` waits...

Reply:
Yes, it is possible. However it's being masked by another issue with the way references are handled in the way the MVP pattern has been implemented.

It's something I plan to fix however I'd rather get the fix in this PR merged as users will almost certainly encounter this if they try and use it in the current state.

Reply:
Okay. Let's merge this and figure out the weird layout issues later.